variables.tf

```diff
@@ -233,3 +231,43 @@ variable "cert_manager" {
     })), null)
   })
 }
+
+variable "fluent_bit" {
+  description = "fluent bit configuration"
+  type = object({
+    enabled      = optional(bool, true)
+    logs_storage = optional(string, "loki")
+    loki = optional(object({
+      tenant_id         = optional(string, "default")
+      logs_endpoint_url = optional(string, null)
+      basic_auth = optional(object({
+        enabled  = optional(bool, false)
+        username = optional(string, null)
+        password = optional(string, null)
+      }), {})
+      bearer_token = optional(object({
+        enabled = optional(bool, false)
+        token   = optional(string, null)
+      }), {})
+    }), {})
+    elasticsearch = optional(object({
+      auth = optional(object({
+        enabled  = optional(bool, false)
+        username = optional(string, null)
+        password = optional(string, null)
+      }), {})
+    }), {})
+    use_defaults = optional(object({
+      outputs = optional(bool, true)
+      filters = optional(bool, true)
+      inputs  = optional(bool, true)
+    }), {})
+    logs_custom = optional(object({
+      outputs = optional(map(string), {})
+      filters = optional(map(string), {})
+      inputs  = optional(map(string), {})
+    }), {})
+    logs_endpoint_url = optional(string, null)
+  })
+  default = {}
```

Comment:
Isn't at least `enabled` needed in defaults to prevent failure in the module's count?